### PR TITLE
Use string relevance for maps

### PR DESCRIPTION
**Pull request by b2ddg**
Adding back string based relevancy check for maps. This used to be the relevancy metric before the single point cutoff was put in place.

@bsstoner @andrey-p

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/spice/maps/maps/maps_maps.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ DDG.require('maps',function(){
         // Mapbox sends back a bunch of places, just want the first one for now
         response = response.features[0];
 
-        if (response.relevance < 0.9) {
+        if (!DDG.isRelevant(response.place_name.toLowerCase()) && response.relevance < 0.9) {
```
Comment (bsstoner) [inline share/spice/maps/maps/maps_maps.js]: Is it worth validating that `response.place_name` exists on the response object before calling toLowerCase() on it?

Comment (bsstoner) [inline share/spice/maps/maps/maps_maps.js]: Is it also worth adding skip words? i.e. `directions`, `map`, etc.?

**Comment by bsstoner** [comment]
:+1:

**Comment by bsstoner** [comment]
:+1:

**Pull request merged**